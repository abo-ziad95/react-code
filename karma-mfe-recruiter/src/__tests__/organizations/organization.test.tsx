import React from 'react';
import renderer from 'react-test-renderer';
import Karma from "@hatech/karma-core";
import Organization from "../../views/organizations/organization";

it("Organization Info renders correctly", () => {
    const organization = renderer
        .create(<Karma><Organization/></Karma>)
        .toJSON();
    expect(organization).toMatchSnapshot();
});