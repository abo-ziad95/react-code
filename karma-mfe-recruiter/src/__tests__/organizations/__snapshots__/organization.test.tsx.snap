// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Organization Info renders correctly 1`] = `
<div
  className="wrapper"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
    >
      <nav
        className="MuiTypography-root MuiBreadcrumbs-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        <ol
          className="MuiBreadcrumbs-ol"
        >
          <li
            className="MuiBreadcrumbs-li"
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
            >
              Organizations
            </a>
          </li>
          <li
            aria-hidden={true}
            className="PrivateBreadcrumbSeparator-root-144 MuiBreadcrumbs-separator"
          >
            /
          </li>
          <li
            className="MuiBreadcrumbs-li"
          >
            <div
              className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
              style={
                Object {
                  "height": undefined,
                  "width": undefined,
                }
              }
            />
          </li>
        </ol>
      </nav>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
      >
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content"
          >
            <span
              className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </span>
          </div>
          <div
            className="MuiCardHeader-action"
          >
            <div
              className="MuiSkeleton-root MuiSkeleton-circle MuiSkeleton-animate"
              style={
                Object {
                  "height": 48,
                  "marginLeft": 12,
                  "width": 48,
                }
              }
            />
          </div>
        </div>
        <hr
          className="MuiDivider-root"
        />
        <div
          className="MuiCardContent-root"
        >
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
          >
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 MuiGrid-grid-sm-2 MuiGrid-grid-md-1 MuiGrid-grid-lg-1 MuiGrid-grid-xl-1"
            >
              Address:
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8 MuiGrid-grid-sm-10 MuiGrid-grid-md-11 MuiGrid-grid-lg-11 MuiGrid-grid-xl-11"
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </div>
          </div>
          <hr
            className="MuiDivider-root"
          />
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
          >
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 MuiGrid-grid-sm-2 MuiGrid-grid-md-1 MuiGrid-grid-lg-1 MuiGrid-grid-xl-1"
            >
              City:
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8 MuiGrid-grid-sm-10 MuiGrid-grid-md-11 MuiGrid-grid-lg-11 MuiGrid-grid-xl-11"
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </div>
          </div>
          <hr
            className="MuiDivider-root"
          />
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
          >
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 MuiGrid-grid-sm-2 MuiGrid-grid-md-1 MuiGrid-grid-lg-1 MuiGrid-grid-xl-1"
            >
              State:
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8 MuiGrid-grid-sm-10 MuiGrid-grid-md-11 MuiGrid-grid-lg-11 MuiGrid-grid-xl-11"
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </div>
          </div>
          <hr
            className="MuiDivider-root"
          />
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
          >
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 MuiGrid-grid-sm-2 MuiGrid-grid-md-1 MuiGrid-grid-lg-1 MuiGrid-grid-xl-1"
            >
              ZIP:
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8 MuiGrid-grid-sm-10 MuiGrid-grid-md-11 MuiGrid-grid-lg-11 MuiGrid-grid-xl-11"
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </div>
          </div>
        </div>
        <hr
          className="MuiDivider-root"
        />
      </div>
    </div>
  </div>
  <hr
    className="MuiDivider-root"
  />
  <div
    className="wrapper-top-bottom"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
    >
      <div
        className="MuiCardHeader-root"
      >
        <div
          className="MuiCardHeader-content"
        >
          <span
            className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
          >
            Transactions
          </span>
          <span
            className="MuiTypography-root MuiCardHeader-subheader MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
          >
            <div
              className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
              style={
                Object {
                  "height": undefined,
                  "width": undefined,
                }
              }
            />
          </span>
        </div>
        <div
          className="MuiCardHeader-action"
        >
          <div
            className="MuiSkeleton-root MuiSkeleton-circle MuiSkeleton-animate"
            style={
              Object {
                "height": 48,
                "marginLeft": 12,
                "width": 48,
              }
            }
          />
        </div>
      </div>
      <hr
        className="MuiDivider-root"
      />
      <ul
        className="MuiList-root MuiList-padding"
      >
        <li
          className="MuiListItem-root MuiListItem-gutters MuiListItem-divider"
          disabled={false}
        >
          <div
            className="MuiListItemText-root"
          >
            <span
              className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </span>
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-gutters MuiListItem-divider"
          disabled={false}
        >
          <div
            className="MuiListItemText-root"
          >
            <span
              className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </span>
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-gutters MuiListItem-divider"
          disabled={false}
        >
          <div
            className="MuiListItemText-root"
          >
            <span
              className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </span>
          </div>
        </li>
      </ul>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item"
      style={
        Object {
          "padding": "8px",
          "textAlign": "center",
        }
      }
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary Mui-disabled Mui-disabled"
        disabled={true}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={-1}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Load More
        </span>
      </button>
    </div>
  </div>
</div>
`;
